<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chatting</class>
 <widget class="QDialog" name="Chatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1041</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TotalLayout">
    <item>
     <layout class="QVBoxLayout" name="Chatting_Layout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>채팅방</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="Chatting_Text">
        <property name="font">
         <font>
          <family>맑은 고딕 Semilight</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Chatting_Layout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="Message_Line">
          <property name="font">
           <font>
            <family>나눔고딕 ExtraBold</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="MessageSend_btn">
          <property name="text">
           <string>전송</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>그룹일정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="GroupScheduleList"/>
      </item>
      <item>
       <widget class="QPushButton" name="GroupScheduleAdd_btn">
        <property name="text">
         <string>그룹 일정 추가</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
